replicator-security/.env

```diff
@@ -0,0 +1 @@
+../utils/config.env
```

Comment:
@sdanduConf can you please clarify why this is required in the `replicator-security` folder (versus just the child folders?)